src/main/java/sirius/web/http/Response.java

```diff
@@ -1369,4 +1370,8 @@ protected void contentionAwareWrite(Object message) {
     public String toString() {
         return "Response for: " + wc.toString();
     }
+
+    private String fetchDebugLevel() {
+        return wc.getCookie(GlobalRenderContext.SIRIUS_DEBUG_COOKIE) != null ? wc.getCookie(GlobalRenderContext.SIRIUS_DEBUG_COOKIE).value() : "OFF";
```

Comment:
`Optional.ofNullable(wc.getCookie(GlobalRenderContext.SIRIUS_DEBUG_COOKIE)).map(Enum::getValue).orElse(DebugLevel.OFF.value());`

Reply:
Value can automatically convert to enums

Reply:
It's actually implemented now as:
`return  Optional.ofNullable(wc.getCookie(GlobalRenderContext.SIRIUS_DEBUG_COOKIE)).map(Cookie::value).orElse("OFF");`